README.md

```diff
@@ -38,7 +38,7 @@ About conda-forge
 
 conda-forge is a community-led conda channel of installable packages.
 In order to provide high-quality builds, the process has been automated into the
-conda-forge GitHub organization. The conda-forge organization contains one repository
+conda-forge GitHub organization. The conda-forge organization contains one repository 
```

Comment:
The canonical source for the README comes from metadata in the recipe. The url there would also need to be updated: https://github.com/conda-forge/m2crypto-feedstock/blob/master/recipe/meta.yaml#L34

Let's hold off on merging this PR until that has been fixed, then the bot will come along and update her PR.